src/presentation/pages/Note.vue

```diff
@@ -1,22 +1,36 @@
 <template>
-  <div v-if="note === null">
-    Loading...
-  </div>
-  <div v-else>
-    <div>
-      <Button
-        v-if="props.id != null"
-        @click="createChildNote"
-      >
-        {{ t('note.createChildNote') }}
-      </Button>
+  <div>
+    <NoteHeader
+      :id="id"
+      :style="{ '--opacity': id ? 1 : 0 }"
```

Comment:
right now it is shown event when not is not loaded. Let's show it only when note is loaded?

Reply:
But then we have "jumping" block of Editor

Without NoteHeader
![image](https://github.com/codex-team/notes.web/assets/94054053/3208efae-f0d2-44e4-811d-273c12abfed7)

With NoteReader
![image](https://github.com/codex-team/notes.web/assets/94054053/662fd075-d7b2-4002-93db-8d3e357658c6)

We have 52 pixels difference

Reply:
If it's not critical I can use v-show to hide it when note is Draft

Reply:
it won't jump because you're hiding via opacity. Just hight it when note is not loaded as well